core/templates/dev/head/pages/library/activity_tiles_infinity_grid_directive.html

```diff
@@ -1,7 +1,33 @@
 <div class="oppia-activities-infinity-grid-container" infinite-scroll="
      showMoreActivities()" infinite-scroll-distance="1"
      infinite-scroll-disabled="endOfPageIsReached || searchResultsAreLoading">
-  <div ng-repeat="activity in allActivitiesInOrder track by $index" style="display: inline-block;">
+  <div ng-repeat="activity in allActivitiesInOrder track by $index" style="display: inline-block;" ng-mouseenter="setHoverOverActivity(activity.id)" ng-mouseleave="setHoverOverActivity(activity.id)">
+    <i class="protractor-test-add-exp-to-playlist-icon oppia-learner-dashboard-icon fa fa-clock-o"
```

Comment:
Shouldn't all these icons just be part of the summary tile? You could, on initialization of the summary tile directive, check if the learner dashboard info service exists, and, if so, show these icons. Or the parent directive can pass in a boolean dictating whether to show these extra things or not.

For example, it doesn't seem unreasonable to add this functionality to summary tiles in the profile page.